conference.js

```diff
@@ -874,6 +875,18 @@ export default {
         APP.store.dispatch(handleToggleVideoMuted(mute, showUI, ensureTrack));
     },
 
+    /**
+     * Simulates blurred background selection/removal on video background. Used by API only.
+     * @param {string} [blurType] Blur type to apply. Accepted values are 'slight-blur', 'blur' or 'none'
+     */
+    setBlurredBackground(blurType) {
+        const tracks = APP.store.getState()['features/base/tracks'];
+        const videoTrack = getLocalVideoTrack(tracks)?.jitsiTrack;
+        const muted = this.isLocalVideoMuted();
+
```

Comment:
You can move this in toggleBlurredBackgroundEffect and dispatch it directly in API.js.

Reply:
Done